modules/checklists/checklists.php

```diff
@@ -959,6 +963,10 @@ public function save_global_checklist()
                 return;
             }
 
+            if (!current_user_can('manage_options')) {
```

Comment:
Considering this method only runs when the user can view the Checklists menu, we have to use the same capability check we used on the menu.

The capability `manage_options` is the default one, but the menu use a filter allowing to customize it. We should use the same approach here:

```
apply_filters('publishpress_checklists_manage_checklist_cap', 'manage_options'),
```

Reply:
Thanks, i've updated this